demos/usage-dag-edges.py

```diff
@@ -16,28 +17,22 @@ def flatten(z):
 server = app.server
 
 # Define a tree in the adjacency list format
-adj_dict = {
-    'root': ['l', 'r'],
-    'l': ['ll', 'lr'],
-    'r': ['rl', 'rr'],
-    'll': ['lll', 'llr'],
-    'rr': ['rrl', 'rrr']
-}
-
-# Flatten values, then only keep the unique entries
-node_ids = list(set(flatten(adj_dict.values()))) + ['root']
-
-nodes = [{'data': {'id': node_id}} for node_id in node_ids]
-edges = flatten([
-    [
-        {'data': {'source': src, 'target': tar}}
-        for tar in adj_dict[src]
-    ]
-    for src in adj_dict
+tree = Tree('a', [
```

Comment:
Nice! This syntax looks a lot easier to understand intuitively :)

Reply:
:)